deployments/models.py

```diff
@@ -260,36 +259,64 @@ class ProgrammeTypes(models.IntegerChoices):
     DOMESTIC = 2, _('Domestic')
 
 
-class Sectors(models.IntegerChoices):
-    WASH = 0, _('WASH')
-    PGI = 1, _('PGI')
-    CEA = 2, _('CEA')
-    MIGRATION = 3, _('Migration')
-    HEALTH = 4, _('Health')
-    DRR = 5, _('DRR')
-    SHELTER = 6, _('Shelter')
-    NS_STRENGTHENING = 7, _('NS Strengthening')
-    EDUCATION = 8, _('Education')
-    LIVELIHOODS_AND_BASIC_NEEDS = 9, _('Livelihoods and basic needs')
-
-
-class SectorTags(models.IntegerChoices):
-    WASH = 0, _('WASH')
-    PGI = 1, _('PGI')
-    CEA = 2, _('CEA')
-    MIGRATION = 3, _('Migration')
-    DRR = 5, _('DRR')
-    SHELTER = 6, _('Shelter')
-    NS_STRENGTHENING = 7, _('NS Strengthening')
-    EDUCATION = 8, _('Education')
-    LIVELIHOODS_AND_BASIC_NEEDS = 9, _('Livelihoods and basic needs')
-    RECOVERY = 10, _('Recovery')
-    INTERNAL_DISPLACEMENT = 11, _('Internal displacement')
-    HEALTH_PUBLIC = 4, _('Health (public)')
-    HEALTH_CLINICAL = 12, _('Health (clinical)')
-    COVID_19 = 13, _('COVID-19')
+@reversion.register()
+class Sector(models.Model):
+    title = models.CharField(max_length=255, verbose_name=_('title'))
+    slug = models.CharField(max_length=255, verbose_name=_('slug'))
```

Comment:
In tests it can be useful, e.g.
primary_sector=Sector.objects.get(slug='WASH'),
at least for the "old classical" sectors.

Reply:
If it's just for testing, using variables should be fine. It's more atomic and easier to use as well.
```python
sector_wash = SectorFactory.create(title='WASH')
# Or just
sector_wash = SectorFactory.create()
```

Reply:
Shall we drop slug?
We can if it seems to be not good.
(And maybe introduce "color" and "deprecated", as it was mentioned on check-in call. Maybe there will be ones which are not used in the future, but kept from past. And the color: instead of the frontend static constant list from here: https://github.com/IFRCGo/go-frontend/blob/develop/src/root/utils/constants.js#L23-L170 )

Reply:
I would suggest dropping it for now.
Makes sense to add color and deprecated if required.

Reply:
Can I drop also "order"? We do not use it, and rendundant near "id".